lua/core/utils.lua

```diff
@@ -154,7 +154,7 @@ M.load_config = function(reload)
 
    local default_config = "default_config"
    local config_name = vim.g.nvchad_user_config or "chadrc"
-   local config_file = vim.fn.stdpath "config" .. "/lua/" .. config_name .. ".lua"
+   local config_file = vim.g.project_config .. "/lua/" .. config_name .. ".lua"
```

Comment:
It's good to catch paths like this,
there may be some paths in `NvChad/extensions` as well 🤔

Reply:
Yes.  I have PR1 there